contracts/multisig/src/key.rs

```diff
@@ -391,12 +395,12 @@ mod ecdsa_tests {
             .unwrap();
         let message = MsgToSign::try_from(ecdsa_test_data::message()).unwrap();
         let public_key = PublicKey::try_from((KeyType::Ecdsa, ecdsa_test_data::pub_key())).unwrap();
-        let result = signature.verify(&message, &public_key).unwrap();
-        assert_eq!(result, true);
+        let result = signature.verify(&message, &public_key);
+        assert_eq!(result.unwrap(), ());
```

Comment:
```suggestion
        assert!(result.is_ok());
```

Reply:
I'm not fan of `is_ok()` because when the test fails it doesn't say anything about the error, it just say the assertion is not true. I would rather remove the assert and add the unwrap to the function call like this: `signature.verify(&message, &public_key).unwrap();`. Thoughts?